### PR TITLE
test: Add parsing test for version cli args

### PR DESCRIPTION
**Pull request by Jason3S**
## Description
Add Unit tests to verify the command line is parsed as expected.

## Motivation and Context

Needed to check exactly how `yargs` parsed the command line. Related to #569 and #575 

## How Has This Been Tested?

It is only a test.

## Types of changes

- [x] Chore (change that has absolutely no effect on users)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by Jason3S** [comment]
@ghiscoding,

If there isn't some strange side-effect I was not expected, I don't believe the errors have anything to do with me adding some more unit test. Might be related to this being a PR from a fork.

**Comment by Jason3S** [comment]
@ghiscoding,

This PR was only to demonstrate how I tested my assumptions about how the command line was parsed.

**Comment by ghiscoding** [comment]
yeah I saw that too, I'm not sure why this is happening considering that `version` is mocked to return `null` and that is why I added my unit tests directly in the `version-command.spec.ts` in my other PR. It might be the move to Vitest that caused this, I don't know. I could take a look later after work. I just got 20min to check some other things over lunch but now it's over... I left a comment & question in the PR I've opened if you could review and reply. Thanks

https://github.com/lerna-lite/lerna-lite/blob/60afb34009b862ea4f3f4a02c80c0c298076c16a/packages/cli/src/cli-commands/__tests__/cli-version-commands.spec.ts#L3-L4

File: packages/cli/src/cli-commands/__tests__/cli-version-commands.spec.ts [hunk 1]
```diff
@@ -2,6 +2,9 @@ import { describe, expect, it, vi } from 'vitest';
 
 vi.mock('@lerna-lite/version', () => null);
```
Comment (ghiscoding) [inline packages/cli/src/cli-commands/__tests__/cli-version-commands.spec.ts:3]: ok I got all your tests working by changing the mock, so change the line 3
```ts
vi.mock('@lerna-lite/version', () => null);
```

with this and that should work
```ts
vi.mock('@lerna-lite/version', () => {
  const VersionCommand = null;
  return { VersionCommand };
});
```
image

**Comment by ghiscoding** [comment]
So I ended up splitting the first PR #575 I had to keep only `conventional-prerelease` code change (without your suggestion) as a new PR #577. If you think that `--dry-run` should still be covered then feel free to add code for that in this PR or a separate PR (if you do, then you probably want to revisit PR #575 to copy over the unit tests I had for it)

**Pull request merged**